src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -186,6 +196,10 @@ const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, o
     [dispatch]
   );
 
+  const onNameChange = useCallback(event => setControlledTextInput({ name: event.target?.value }), [setControlledTextInput]);
```

Comment:
I believe you are missing the spead here:
```suggestion
setControlledTextInput({ ...controlledTextInput, name: event.target?.value })
```